### PR TITLE
Medical items sound support + sounds

### PR DESCRIPTION
**Pull request by elthundercloud**
## About the PR 
Fix:  #6786
Medical items have  sounds:
- Medical items have `healingBeginSound` and `healingEndSound` data  fields
- Ointment, brutepack and gauze now have basic sounds I stole from various places
- Healing items now have `BaseHealingItem` as parent, because it's easier

### CODE REVIEW

File: Content.Server/Medical/HealingSystem.cs [hunk 1]
```diff
@@ -5,16 +5,20 @@
...
 using Content.Shared.Interaction.Helpers;
 using Content.Shared.Stacks;
+using Robust.Shared.Audio;
+using Robust.Shared.Player;
 
 namespace Content.Server.Medical;
 
 public sealed class HealingSystem : EntitySystem
 {
+    [Dependency] private readonly ActionBlockerSystem _blocker = default!;
```
Comment (metalgearsloth) [inline Content.Server/Medical/HealingSystem.cs:21]: ?

File: Content.Server/Medical/HealingSystem.cs [hunk 2]
```diff
@@ -56,6 +60,11 @@ private void OnHealingComplete(EntityUid uid, DamageableComponent component, Hea
             _logs.Add(LogType.Healed, $"{EntityManager.ToPrettyString(args.User):user} healed {EntityManager.ToPrettyString(uid):target} for {healed.Total:damage} damage");
         else
             _logs.Add(LogType.Healed, $"{EntityManager.ToPrettyString(args.User):user} healed themselves for {healed.Total:damage} damage");
+
+        if (args.Component.HealingEndSound != null)
+        {
+            SoundSystem.Play(Filter.Pvs(uid), args.Component.HealingEndSound.GetSound(), uid, AudioHelpers.WithVariation(0.125f).WithVolume(-5f));
```
Comment (metalgearsloth) [inline Content.Server/Medical/HealingSystem.cs:66]: Pass in EntityManager to the Pvs static methods.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Licensing for audio?

**Comment by elthundercloud** [comment]
>Licensing for audio?
Freesound.org and I slapped it together in Reaper

**Comment by Willhelm53** [comment]
Good work! You'll need to include a quick licenses.txt, see Resources/Audio/Animals for an example.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**